app/controllers/reports_controller.rb

```diff
@@ -39,9 +39,9 @@ def create
     raise "No background specified and background #{Background::DEFAULT_BACKGROUND_MODEL_NAME} does not exist" unless @report.background
     total_reads = @report.pipeline_output.total_reads
     summary = @report.background.summarize
-    data = @report.pipeline_output.taxon_counts.map { |h| h.attributes.merge(norm_count: h["count"] / total_reads.to_f) }
```

Comment:
I think all the logic here probably should be moved to Report model. But you can do that in the next refactoring.

Reply:
agree